### PR TITLE
JAMES-3775 Add Wait for healthcheck to ClaimAV/RspamD extension

### PR DESCRIPTION
**Pull request by vttranlina**

### CODE REVIEW

**Review by quantranhong1999 — APPROVED** [review]
Well `clamav` and `rspamd` docker images do support HEALTHCHECK.

Maybe one day we should add Docker HEALTHCHECK to James image?

**Review by Arsnael — APPROVED** [review]

**Comment by vttranlina** [comment]
> Maybe one day we should add Docker HEALTHCHECK to James image?

How we can do it?
James has multiple modules/ several servers, it depends on configuration. Ex we health check Jmap server, but we disable jmap in conf, so then health check is not available.
If we use the log for healthcheck, it depends on logback configuration file

**Comment by chibenwa** [comment]
Healthcheck is held by webadmin.

We do turn it on by defaut in docker as it requires port binding to be exposed.

Guice lifecycle health check seems like a goid starting point.

Is the healthcheck feature supported by jib?

**Comment by quantranhong1999** [comment]
> Is the healthcheck feature supported by jib?

Seems no.

TLDR: Healthcheck is a Docker container feature specific rather than a common Open Container Initiative standard. And JIB won't support Healthcheck until it is an OCI spec.
rf: https://github.com/GoogleContainerTools/jib/issues/676
